src/features/CellEditing.js

```diff
@@ -3,12 +3,39 @@
 var Feature = require('./Feature');
 var CellEditor = require('../cellEditors/CellEditor');
 
+var KEYS = {
+    RETURN: 'RETURN',
+    RETURNSHIFT: 'RETURNSHIFT',
+    DELETE: 'DELETE',
+    BACKSPACE: 'BACKSPACE',
+    SPACE: 'SPACE',
+    F2: 'F2'
+};
+
 /**
  * @constructor
  * @extends Feature
  */
 var CellEditing = Feature.extend('CellEditing', {
 
+    isVisibleChar: function(char, event) {
+        return !!(
+            (char.length === 1) && !(event.detail.meta || event.detail.ctrl)
+        );
+    },
+
+    isSpaceChar: function(char, event) {
+        return char === KEYS.SPACE;
+    },
+
+    isDeleteChar: function(char, event) {
+        return (char === KEYS.DELETE || char === KEYS.BACKSPACE);
+    },
+
+    isEditChar: function(char, event) {
+        return (char === KEYS.F2 || char === KEYS.RETURN || char === KEYS.RETURNSHIFT);
```

Comment:
including RETURN here opens the editor but short circuits current functionality which is that RETURN goes down one cell and SHIFT-RETURN goes up one cell (as down- and up-arrrow, respectively). Let's remove these from here. Developer could override this method to get this functionality.